avocado/core/runner.py

```diff
@@ -428,38 +428,43 @@ def run_suite(self, test_suite, mux, timeout=0, replay_map=None,
 
         index = -1
         for test_template in test_suite:
```

Comment:
yep :)
Trying to understand the travis issue before send a new version with the `try` outside the `for`.